src/OpenColorIO/Baker.cpp

```diff
@@ -388,6 +395,16 @@ OCIO_ADD_TEST(Baker_Unit_Tests, bake)
 
 }
 
+OCIO_ADD_TEST(Baker_Unit_Tests, bake_empty_config)
+{
+    // Verify that running bake with an empty configuration
+    // throws an exception and does not segfault.
+    OCIO::BakerRcPtr bake = OCIO::Baker::Create();
+    bake->setFormat("cinespace");
```

Comment:
[Request] The `setFormat()` method does not validate that the format accepts the 'bake' mode.  That could be something interesting to add if you have time.

Reply:
Added the validation.